spec/std/file_spec.cr

```diff
@@ -313,7 +314,7 @@ describe "File" do
     File.extname("").should eq("")
   end
 
-  it "constructs a path from parts" do
+  pending_win32 "constructs a path from parts" do
```

Comment:
I would actually remove these specs because this is already tested extensively in `path_spec`.
One example can be kept to ensure `File.join` is working, but that would be a simple one that doesn't have any platform-specific differences.
```cr
it ".join" do
  File.join(["/foo/", "/bar"]).should eq("/foo/bar")
end
```
Ditto below for `.expand_path`. 
I can see this being a subsequent PR, if you don't want to tackle this directly. Inserting a TODO comment wouldn't hurth, though.